src/iomusxml.cpp

```diff
@@ -2193,6 +2187,14 @@ void MusicXmlInput::ReadMusicXmlNote(
             fTrem->SetBeams(beamFloatNum + beamAttachedNum);
             fTrem->SetBeamsFloat(beamFloatNum);
         }
+        else if (!HasAttributeWithValue(tremolo.node(), "type", "stop")) {
```

Comment:
Shouldn't that be `"type", "single"`?

Reply:
No. 
1. type attribute is not required (backward compatibility to MusicXML 1.1) and 
2. MusicXML 3.1 brings `type="unmeasured"`.

So there are three possibilities to encode a MEI `bTrem` in MusicXML.
See https://www.musicxml.com/for-developers/version-history/
(and https://github.com/w3c/musicxml/pull/200/files)

Reply:
Ah, I have not looked at the version history, just at the manual: http://usermanuals.musicxml.com/MusicXML/MusicXML.htm#EL-MusicXML-tremolo.htm, but apparently that's not the latest info.

... and I have missed the `!`sign; sorry for that. I understand now. Thanks for clarification!